Index: mfo
===================================================================
--- mfo	(revision 22443)
+++ mfo	(working copy)
@@ -3,7 +3,7 @@
 # https://gist.github.com/JBlond/2fea43a3049b38287e5e9cefc87b2124
 declare -rA color_text=(
                   [black]='\e[0;30m'
-             [bold-black]='\e[1;30m'
+             [bold-black]='\e[1;30m's
         [underline-black]='\e[4;30m'
           [intense-black]='\e[0;90m'
      [intense-bold-black]='\e[1;90m'