diff --git a/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriReceptionMonitorHandlerVduTests.cpp b/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriReceptionMonitorHandlerVduTests.cpp
index 067bdff8cc..b9ebf5c227 100755
--- a/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriReceptionMonitorHandlerVduTests.cpp
+++ b/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriReceptionMonitorHandlerVduTests.cpp
@@ -13,7 +13,7 @@
 #include <lim/storage_data/pddb_types/SUPPORT_CHANNEL.hpp>
 #include <lim/storage_data/pddb_types/CELL_M.hpp>
 
-#include <pm/agent/def.hpp>
+#include <pm/agent/def.hspp>
 #include <pm/uri/uri.h>
 
 #include <Multiradio_def.h>
diff --git a/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriTechLogHandlerTests.cpp b/SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriTechLogHandlerTests1.cpp
similarity index 100%
rename from SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriTechLogHandlerTests.cpp
rename to SC_MONOLITH/DM_RUMAG/test/UT/tests/ECpri/PerformanceMonitoringManagement/ECpriTechLogHandlerTests1.cpp
diff --git a/django.txt b/django.txt
new file mode 100644
index 0000000000..e18c37483a
--- /dev/null
+++ b/django.txt
@@ -0,0 +1 @@
+just for test
\ No newline at end of file
diff --git a/test.txt b/test.txt
deleted file mode 100644
index 74942f2320..0000000000
--- a/test.txt
+++ /dev/null
@@ -1 +0,0 @@
-dummy commit
